Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -36,6 +36,13 @@ public protocol ProtobufBinaryCodableType: ProtobufTypePropertiesBase {
 
     // Special interface for decoding a value of this type as a map value.
     static func decodeProtobufMapValue(decoder: inout ProtobufFieldDecoder, value: inout BaseType?) throws
+
+    /// Returns the number of byte required to encode `value` on the wire.
+    ///
+    /// Note that for length-delimited data (such as strings, bytes, and messages), the returned
+    /// size includes the space required for the length prefix. For messages, this is a subtle
+    /// distinction from the `serializedSize()` method, which does *not* include the length prefix.
+    static func encodedSize(of value: BaseType) throws -> Int
```

Comment:
The other protobuf implementations fairly consistently use some form of `NoTag` in the names to describe this (i.e., the size of whatever would come after the tag, including the length if present), so in the interest of improving a reader's ability to map concepts between the Swift library and others, maybe `encodedSizeWithoutTag(of:)` would be best.